supports/mover-performance-test/prepare.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+. ./config
+
+# generate test data using DFSIO
+for size in "${!CASES[@]}"; do
+    num=${CASES[$size]}
+    dir="${size}_${num}"
+    ssh ${SRC_NODE} "hdfs dfs -mkdir -p /${dir}/io_data"
+    hadoop jar $HADOOP_HOME/hadoop-mapreduce/hadoop-mapreduce-client-jobclient-*-tests.jar TestDFSIO -write -nrFiles $(($num)) -size ${size}  -resFile /home/hdfs/dfsio_mover_test.log
```

Comment:
The log path is env. dependent. I recommend to use `$(pwd)/dfsio_mover_test.log`.